### PR TITLE
Feature/root entry point

### PR DESCRIPTION
**Pull request by hutnikau**
require:
https://github.com/oat-sa/tao-core/pull/964

In log are a lot of such records:
`2016-08-18 15:24:13 [ERROR] 'Extension or controller not found' /var/www/html/tao/models/classes/mvc/error/ResponseAbstract.php 122`

It occurs each time after request to the root of Tao. Also after such request a user redirects to the login form even if he has already logged in. This PR adds route for root of the Tao to redirect logged in users to entry page and anonymous users to login form and gets rid of a thousands of error messages in the log.

### CODE REVIEW

**Comment by bugalot** [comment]
@hutnikau , It seems fine to me, but I also would like to have the point of view of @jbout on this.
Thanks for this proactive resolution, that's great to remote such a polluting log entry. +1 !

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -100,6 +100,16 @@ public function update($initialVersion)
         }
         
         $this->skip('1.2.2', '1.6.1');
+
+        if ($this->isVersion('1.6.1')) {
+            $anonymousUserRole = new \core_kernel_classes_Resource(INSTANCE_ROLE_ANONYMOUS);
```
Comment (bugalot) [inline scripts/update/Updater.php]: Hello @hutnikau , could you just please make use of the ACLProxy in the Update script?
Thanks a lot !

File: manifest.php [hunk 2]
```diff
@@ -43,7 +43,9 @@
     'managementRole' => 'http://www.tao.lu/Ontologies/generis.rdf#taoCeManager',
     'acl' => array(
         array('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BackOfficeRole', array('ext' => 'taoCe', 'mod' => 'Main', 'act' => 'index')),
-        array('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BackOfficeRole', array('ext' => 'taoCe', 'mod' => 'Home'))
+        array('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BackOfficeRole', array('ext' => 'taoCe', 'mod' => 'Home')),
+        array('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BaseUserRole', array('ext'=>'taoCe', 'mod' => 'Main', 'act' => 'rootEntry')),
```
Comment (bugalot) [inline manifest.php]: Actually, BaseUserRole includes AnonymousRole. So basically, I think there is no necessity to explicitly grand the BaseUserRole, but just the AnonymousRoule. Indeed, BaseUserRole will implicitly inherits AnymousRole grants.

File: scripts/update/Updater.php [hunk 3]
```diff
@@ -100,6 +102,16 @@ public function update($initialVersion)
         }
         
         $this->skip('1.2.2', '1.6.1');
+
+        if ($this->isVersion('1.6.1')) {
+
+            AclProxy::applyRule(new AccessRule('grant', INSTANCE_ROLE_ANONYMOUS,
+                ['ext'=>'taoCe', 'mod' => 'Main', 'act' => 'rootEntry']));
+            AclProxy::applyRule(new AccessRule('grant', INSTANCE_ROLE_BASEUSER,
```
Comment (bugalot) [inline scripts/update/Updater.php]: Same remark as above, 

"Actually, BaseUserRole includes AnonymousRole. So basically, I think there is no necessity to explicitly grand the BaseUserRole, but just the AnonymousRoule. Indeed, BaseUserRole will implicitly inherits AnymousRole grants."

**Comment by hutnikau** [comment]
I have resolved conflicts and made changes according to your comments. 
Please review it again.

**Pull request merged**